assets/src/scss/pages/post/_post.scss

```diff
@@ -1,5 +1,5 @@
 .single-post,
-.single-attachment {
```

Comment:
As we [removed](https://jira.greenpeace.org/browse/PLANET-5908) the comment section from attachments view page, hence removing the `single-attachment` class and added the `single-idea`

Reply:
It was leftover from [this](https://github.com/greenpeace/planet4-master-theme/pull/1287/files) PR.